### DISCUSSION TITLE
Dokku beginner: Why does my Rails app not have tables?

### DISCUSSION THREAD

**Discussion by pessi-v** [dokku/dokku#6900] [Q&A]
I've deployed a fresh Rails 7.1 app, without Dockerfile, so I guess with Heroku buildpacks as set up by the bootstrap script, and I could see the migrations running without errors. However, the table for my only model is not there.

Is there a step I'm missing in Dokku setup? I tried creating a storage with 
`dokku storage:ensure-directory my-app`
`dokku storage:mount my-app /var/lib/dokku/storage/my-app:/storage`

**Comment by josegonzalez** [comment]
Are you sure you aren't accidentally pushing to a different database or something? Persistent storage shouldn't matter here unless you're somehow using sqlite and storing it in /storage (which again, shouldn't matter).
